### PR TITLE
Separate env variables

### PR DESCRIPTION
**Pull request by metemaddar**
Relate issue: #1830 #1831

### CODE REVIEW

File: app/api/src/tests/api/api_v1/list_todos.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+for filename in *.py; do
+    if grep -q '# TODO:' $filename;
+    then
+        echo "### $filename"
+        sed -n -e '/^# TODO:/p' $filename| sed -e "s/^# TODO://" | sed 's/.*/- [ ]&/'
+    fi
+done
```
Comment (majkshkurti) [inline app/api/src/tests/api/api_v1/list_todos.sh:7]: We might not need this. We can use this plugin: 
https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight

Comment (metemaddar) [reply app/api/src/tests/api/api_v1/list_todos.sh:7]: This is not highlighter. It lists the TODOs for github issue. The code highlighters already highlight TODOs.

Comment (majkshkurti) [reply app/api/src/tests/api/api_v1/list_todos.sh:7]: I get it. So you are exporting to copy it when creating the epic issues (markdown). The highlight plugin also can export but not sure if markdown.
Anyways, maybe we can extend this also for the front end, as now this works only for the python files.

Comment (metemaddar) [reply app/api/src/tests/api/api_v1/list_todos.sh:7]: This PR is for separating env variables of docker-compose and uvicorn Ports. In related issues there are more details.

**Pull request merged**